components/dashboard/src/feedback-form/FeedbackModal.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import Modal from "../components/Modal";
+import FeedbackComponent from "./FeedbackComponent";
+
+function FeedbackFormModal(props: { onClose: () => void }) {
+    const onClose = () => {
+        props.onClose();
+    };
+
+    const onSubmit = () => {
+        props.onClose();
+    };
+
+    return (
+        <Modal visible={true} onClose={onClose}>
```

Comment:
**suggestion**: Once we address all the UX comments on the styling of the second step, could we adjust the height of the modal using padding or adjusting the custom textarea height, so that modal has the same height for all steps?

Reply:
Fixes pushed! Do you mean that the first flow with only the emojis should be the same height as the expanded one?

Reply:
Done!

Reply:
Looks great!